Jenkinsfile

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'smart-answers'
+
+node {
+  def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
+
+  try {
+    stage('Checkout') {
+      checkout scm
+      govuk.cleanupGit()
+      govuk.mergeMasterBranch()
+      govuk.contentSchemaDependency()
+      govuk.setEnvar("GOVUK_CONTENT_SCHEMAS_PATH", "tmp/govuk-content-schemas")
+      govuk.setEnvar("DISPLAY", ":99")
+    }
+
+    stage('Bundle') {
+      govuk.bundleApp()
+    }
+
+    stage('Linter') {
+      govuk.rubyLinter()
+    }
+
+    stage("Assets") {
+	    govuk.precompileAssets()
+	  }
+
+    stage('Tests') {
+      govuk.runTests()
+    }
+
+    stage('Push release tag') {
+      govuk.pushTag(REPOSITORY, BRANCH_NAME, 'release_' + BUILD_NUMBER)
+    }
+
+    stage('Deploy to Integration') {
+      // Deploy on Integration (only master)
+      govuk.deployIntegration(REPOSITORY, BRANCH_NAME, 'release', 'deploy')
+    }
```

Comment:
I'd recommend wrapping the 'Push release tag' and 'Deploy to Integration' stages to only run when building the master branch (e.g. like [1]), as this makes the display in Jenkins less surprising.

1: https://github.com/alphagov/publishing-api/blob/master/Jenkinsfile#L55